packages/@lwc/engine-core/src/framework/membrane.ts

```diff
@@ -7,6 +7,8 @@
 import ObservableMembrane from 'observable-membrane';
 import { valueObserved, valueMutated } from './mutation-tracker';
 
+export const tagPropertyKey = Symbol.for('@@lockerLiveValue');
```

Comment:
Nitpick: Outside the context of the membrane the name `tagPropertyKey` doesn't make much sense. Since this property is made for Locker, I think it would make sense to reflect it in the exported identifier name.

```suggestion
export const lockerLivePropertyKey = Symbol.for('@@lockerLiveValue');
```